.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,57 @@
+name: Bug report
+description: Create a bug report to help us improve Prisma
+labels: "kind/bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us improve Prisma! 🙏 Please follow the sections in the template and provide as much information as possible about your problem, e.g. by setting the `DEBUG="*"` environment variable and enabling additional logging output in Prisma Client.
+        Learn more about writing proper bug reports here: https://pris.ly/d/bug-reports
```

```suggestion
        Learn more about the best practice for writing a bug report here: https://pris.ly/d/bug-reports
```

"proper" can be weird I think